{
  "name": "kinvey-datalink-errors",
  "version": "0.3.2",
  "description": "Error mapping for Datalink Connectors",
  "license": "Apache-2.0",
  "repository": {
    "type": "git",
    "url": "Kinvey/kinvey-datalink-errors"
  },
  "engines": {
    "node": "= 0.10.x"
  },
  "devDependencies": {
    "eslint": "4.7.2",
    "eslint-config-kinvey-platform": "kinvey/eslint-kinvey-platform#v0.1.1",
    "eslint-plugin-import": "2.7.0",
    "mocha": "5.2.0",
    "should": "3.3.2"
  },
  "scripts": {
    "pretest": "./node_modules/.bin/eslint lib test",
    "lint": "npm run pretest",
    "test": "npm run test-unit && npm run test-integration",
    "test-unit": "mocha test/unit/*/*",
    "test-integration": "echo \"No integration tests available for the moment\"",
    "test-npm-security": "npm audit"
  }
}
